Mathlib/Data/List/Sort.lean

```diff
@@ -327,6 +327,37 @@ theorem sublist_orderedInsert (x : α) (xs : List α) : xs <+ xs.orderedInsert r
   refine Sublist.trans ?_ (.append_left (.cons _ (.refl _)) _)
   rw [takeWhile_append_dropWhile]
 
+theorem cons_sublist_orderedInsert {l c : List α} {a : α} (hl : c <+ l) (ha : ∀ a' ∈ c, a ≼ a') :
+    a :: c <+ orderedInsert r a l := by
+  induction l with
+  | nil         => simp_all only [sublist_nil, orderedInsert, Sublist.refl]
+  | cons _ _ ih =>
+    unfold orderedInsert
+    split
+    · exact .cons₂ _ hl
+    · cases hl
+      · exact .cons _ <| ih ‹_›
```

Comment:
You're using `‹_›` a lot, which isn't great for readability. I'd suggest switching to `cases ... with` and `split_ifs`, if that helps, and otherwise at least fill in the underscores.

Reply:
Fixed